pom.xml

```diff
@@ -60,7 +60,7 @@
     <spark.version>2.4.0</spark.version>
     <spark.core.artifactid>spark-core_2.11</spark.core.artifactid>
     <spark.sql.artifactid>spark-sql_2.11</spark.sql.artifactid>
-    <htsjdk.version>2.9.1</htsjdk.version>
+    <htsjdk.version>2.10.0</htsjdk.version>
```

Comment:
> I tried going with the latest one but got some other test failure (something with phasing...I didn't bother digging into it for now, saved it for a later pass)
gatk uses htsjdk version `2.23.0`, would that affect this use case?

Reply:
The reason I had to upgrade htsjdk is to ensure that htsjdk returns the contig header lines in original order rather than lexicographic order. That's what this use case requires...and htsjdk has the same behavior in that regard between 2.10.0 and 2.23.0

As you say, htsjdk is one of the packages that GATK excludes from us - they use the latest version anyway. I think the htsjdk issue is (probably) with our test (though maybe its an htsjdk issue...?).

Reply:
As you mention, if its only the haploid triploid test, where one of the samples has mixed phase in the genotypes and the golden output (when querying without genotypes) converts that to `././.`, while with the latest htsjdk it comes out as `.|.|.`, let's wait for @kgururaj's take on this. Maybe, we can change our golden output and move to 2.23.0 as well - as part of another PR.